### PR TITLE
fix: Add fetch assets from route action to Going back router action

### LINKED ISSUE #1674: Going back should fetch results with the previous params.
## 🎉 Description

With the current UX, when a user goes back, the cards are not reloaded but the filters are removed. The expected behavior should be to go back to the previous param configuration and re-fetch the results.

* [X] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #1674

## The bug

https://github.com/decentraland/marketplace/assets/8763687/56df1b9e-604c-4480-94c0-726a635777c4

## The fix

https://github.com/decentraland/marketplace/assets/8763687/89538cac-4df1-419e-8532-61a963f2b666

### CODE REVIEW

File: webapp/src/modules/routing/sagas.ts [hunk 1]
```diff
@@ -70,6 +71,7 @@ import {
   BROWSE,
   BrowseAction,
   FETCH_ASSETS_FROM_ROUTE,
+  fetchAssetsFromRoute as FetchAssetsFromRoute,
```
Comment (kevinszuchet) [inline webapp/src/modules/routing/sagas.ts:74]: I'd rename this to `fetchAssetsFromRouteAction` if possible for standards purposes. WDYT?

**Review by kevinszuchet — DISMISSED** [review]

File: webapp/src/modules/routing/sagas.ts (hunk 1 above)
Comment (juanmahidalgo) [reply webapp/src/modules/routing/sagas.ts]: I was going to use that name but I saw the line from below that the same name. But I guess wthat if if doesn't start with capital letter it's just fine. Updated!

Comment (kevinszuchet) [reply webapp/src/modules/routing/sagas.ts]: Could you rename it also in the tests file? We still have an import with the following alias: `fetchAssetsFromRoute as FetchAssetsFromRouteAction`.

File: webapp/src/modules/routing/sagas.spec.ts [hunk 2]
```diff
@@ -25,7 +31,7 @@ import {
   fetchAssetsFromRoute as FetchAssetsFromRouteAction
```
Comment (kevinszuchet) [inline webapp/src/modules/routing/sagas.spec.ts:31]: I'd also rename this. WDYT?

Comment (juanmahidalgo) [reply webapp/src/modules/routing/sagas.spec.ts]: sure, done 👍

**Review by kevinszuchet — APPROVED** [review]

**Pull request merged**